soccer/gameplay/robocup-py.cpp

```diff
@@ -958,9 +929,8 @@ BOOST_PYTHON_MODULE(robocup) {
         .def("eval_pt_to_our_goal", &KickEval_eval_pt_to_our_goal)
         .def("eval_pt_to_seg", &KickEval_eval_pt_to_seg);
 
-    class_<std::function<float(Geometry2d::Point)>,
-           std::function<float(Geometry2d::Point)>*>("stdfunction", no_init)
-        .def("__init__", make_constructor(&stdfunction_constructor));
+    class_<PythonFunctionWrapper>("PythonFunctionWrapper")
+        .def("get_function", &PythonFunctionWrapper::getFunction);
```

Comment:
This line is now complaining about missing arguments to the `PythonFunctionWrapper` constructor. I think you need to define a similar constructor like `NelderMead2DConfig`.